x11/xedit/files/apple-silicon.diff

```diff
@@ -0,0 +1,52 @@
+--- lisp/math.c.orig	2020-09-06 21:49:04.000000000 -0400
++++ lisp/math.c	2020-09-06 21:50:16.000000000 -0400
+@@ -32,9 +32,7 @@
+ #include "lisp/math.h"
+ #include "lisp/private.h"
+ 
+-#ifdef __UNIXOS2__
+-# define finite(x) isfinite(x)
+-#endif
++#define finite(x) isfinite(x)
```

Comment:
Won't this conflict with the existing definition of `finite` on Intel systems?

Reply:
If you changed the patch so that it only changed the `#ifdef __UNIXOS2__` lines to something like
```
#if defined(__UNIXOS2__) || defined(__arm64__)
```
then the patch could be applied unconditionally which would be preferable.

Reply:
I have updated the patch and made it unconditional as suggested above.